integration-auth/src/main/java/org/symphonyoss/integration/authentication/jwt/JwtAuthentication.java

```diff
@@ -33,10 +50,20 @@
 
   public static final String AUTHORIZATION_HEADER_PREFIX = "Bearer ";
 
+  private static final int CACHE_EXPIRATION_IN_MINUTES = 60;
```

Comment:
I'm not sure what's the best number here. We can use this number or greater than it as default value and define a parameter on the YAML config file.